COM-413 Implement testing indicator 3b

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COM-413

tmvumbi2 committed:
COM-413 Implement testing indicator 3b
Implements testing indicator 3b

tmvumbi2 committed:
Merge branch 'dev' into COM-413

# Conflicts:
#	reportssql/indicator_functions.sql
#	reportssql/testing_report_util_functions.sql

tmvumbi2 committed:
COM-413 Fix indicator 3b function
Removes the call to patientAgeAtReportEndDateIsBetween: the function patientAgeAtVirologicHIVTestIsBetween is already taking care of desagregation

Removes the call to patientHadAVirologicHIVTestDuringReportingPeriod: this condition is already included in patientHadAPositiveVirologicHIVTestResultDuringReportingPeriod

Replaces the calls to patientWasEnrolledToHIVProgramBeforeVirologicTest, patientWasInitiatedToARVBeforeVirologicTest and patientTakingARVAtDateOfVirologicTest by existing functions

lauravignoli commented on reportssql/testing_report_util_functions.sql:
```diff
@@ -567,6 +567,7 @@ BEGIN
         RETURN dateAtVirologicHIVTestLabForm;
     END IF;
 
+    RETURN (dateAtVirologicHIVTest);
```
@tmvumbi2 This is not necessary

tmvumbi2 committed:
COM-413 Remove RETURN (dateAtVirologicHIVTest) from getDateOfVirologicTest
The above return statement is incorrect and not needed (it was left there by mistake)

lauravignoli committed:
COM-413 Fix patientHadAPositiveVirologicHIVTestResultDuringReportingPeriod

A obs voided check was missing.
